`fprime-util new` not working since v3.1.0

| | |
|:---|:---|
|**_F´ Version_**| fprime-tools >=3.1.0 |
|**_Affected Component_**| `fprime-util new`  |
---
## Problem Description

fptools v3.1.0 added the `pass_through` argument to the `runners[parsed.command]` call for calling the runner (see [diff](https://github.com/fprime-community/fprime-tools/compare/v3.0.1...v3.1.0#diff-d43bbde2dc67ba20cd1cbb14f1e42f9faf959482a8beb1b0ef690f5331c7f797R253)). However the callback for the `new` runner was not updated accordingly, and is still only taking 4 arguments instead of 5 (see [here](https://github.com/fprime-community/fprime-tools/blob/v3.1.1/src/fprime/util/cli.py#L106))

## How to Reproduce

1. `$ pip install fprime-tools==3.1.0` (or any version >=3.1.0)
2. `$ fprime-util new --component` throws 
```
[ERROR] template() takes 4 positional arguments but 5 were given
```

## Comment
Has this not been caught because this command is not used with F` v3, since it's generating XML? The fix is very easy so I'll make a PR. Should help with https://github.com/fprime-community/fprime-tools/issues/37.

--- a/src/fprime/util/cli.py
+++ b/src/fprime/util/cli.py
@@ -103,7 +103,7 @@ def hash_to_file(
 
 
 def template(
-    build: Build, parsed: argparse.Namespace, _: Dict[str, str], __: Dict[str, str]
+    build: Build, parsed: argparse.Namespace, _: Dict[str, str], __: Dict[str, str], ___
 ):
     """Processes new command
 
@@ -112,6 +112,7 @@ def template(
         parsed: parsed arguments
         _: unused cmake arguments
         __: unused make arguments
+        ___: unused pass through arguments
     """
     if parsed.component and not parsed.port:
         return new_component(build.deployment, parsed.platform, parsed.verbose, build)

